### PR TITLE
Task/improve hardcoded styles and types

### PR DESCRIPTION
**Pull request by jfrader**
I revamped the entire messages,  contacts and sellout apps to use their own theme provider, merged with global one.

This should be the way IMO to make any custom app colors.

Also I improved the way os/useApps worked and
The way useKeyboard works (removed ugly event from Phone.tsx)

### CODE REVIEW

**Comment by RockySouthpaw** [comment]
Will review this in a few hours :)

**Review by RockySouthpaw — COMMENTED** [review]
Tested in game as of this comment, everything worked fine and the UI improvements all look well.

Someone will need to review the code.

**Comment by jfrader** [comment]
> Tested in game as of this comment, everything worked fine and the UI improvements all look well.
> 
> Someone will need to review the code.

@itschip @TasoOneAsia

File: phone/src/apps/messages/components/modal/MessageBubble.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    maxWidth: '80%',
+    height: '80%',
+    maxHeight: '80%',
+  },
+  mySms: {
+    float: 'right',
+    margin: theme.spacing(1),
+    padding: '6px 12px',
+    height: 'auto',
+    width: '80%',
```
Comment (itschip) [inline phone/src/apps/messages/components/modal/MessageBubble.tsx:17]: Let's set a min and max width on the papers, so the message 'bubble' doesn't stretch so much as it does now,

File: phone/src/apps/messages/components/modal/MessageBubble.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    background: theme.palette.background.default,
+    color: theme.palette.text.primary,
+    borderRadius: '20px',
+    textOverflow: 'ellipsis',
+  },
+  sms: {
+    float: 'left',
+    margin: theme.spacing(1),
+    padding: '6px 12px',
+    width: '80%',
```
Comment (itschip) [inline phone/src/apps/messages/components/modal/MessageBubble.tsx:27]: Let's set a min and max width on the papers, so the message 'bubble' doesn't stretch so much as it does now,

File: phone/src/apps/messages/components/modal/MessageBubble.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            <img
+              src={message.message}
+              className={classes.imageMessage}
+              alt='multimedia'
+            />
+          ) : (
+            <div>{message.message}</div>
+          )}
+        </Box>
+        <Box textAlign='right'>
```
Comment (itschip) [inline phone/src/apps/messages/components/modal/MessageBubble.tsx:65]: Remove the styling on the Box.

**Review by itschip — CHANGES_REQUESTED** [review]

File: phone/src/apps/messages/components/modal/MessageBubble.tsx (hunk 3 above)
Comment (jfrader) [reply phone/src/apps/messages/components/modal/MessageBubble.tsx:65]: Why?

**Comment by RockySouthpaw** [comment]
Sellout looks a lot better now with other themes so good work on that. 

Alerts stilll have an overflow issue where if the number exceeds a certain amount it will overflow essentially. Not sure if this is out side the scope of this PR though. Error [here](https://app.gitkraken.com/glo/card/1235577709d04f95be80db677c6b8a1b)

This PR also fixes the `update` and `delete` buttons not being properly alligned anymore.
[Before(https://media.discordapp.net/attachments/791854455468458042/812081641362685952/unknown.png) [Now](https://i.imgur.com/DMw5PCR.png)

**Review by itschip — APPROVED** [review]
LGTM 👍

**Pull request merged**